eo/notebook_config.yml.template

```diff
@@ -0,0 +1,10 @@
+# Config file containing the list of notebooks to download for a specific image.
+#
+# Used in the deploy-data-specific-image script from birdhouse-deploy.
+# More details on this config can be found on this script file :
+#   https://github.com/bird-house/birdhouse-deploy/blob/master/birdhouse/deployment/deploy-data-specific-image
+
+- repo_url: https://github.com/crim-ca/pavics-jupyter-images
+  branch: master
+  source_path: eo/notebooks
+  dest_dir: ${JUPYTERHUB_USER_DATA_DIR}/tutorial-notebooks/eo-crim
```

Comment:
You see here, what could prevent an image maintainer to cause trouble and set the dest_dir to ${JUPYTERHUB_USER_DATA_DIR}? (See : https://github.com/bird-house/birdhouse-deploy/pull/150#discussion_r616723622)